Example/Example/ContentView.swift

```diff
@@ -6,7 +6,7 @@ struct ContentView: View {
     private var userId = "123"
 
     @State
-    private var appName = ""
+    private var appName = "Babbel"
```

Comment:
Perhaps 
```suggestion
    private var appName = "Example"
```
or 

```suggestion
    private var appName = "Daily Wisdom"
```

to align more with the rest of the mock example app and not introduce confusion?